README.md

```diff
@@ -1,288 +1,278 @@
-
-<p align="center">
-  <div style="text-align: center;">
-    <a href="https://cloud.tembo.io/sign-up">
-      <img src="https://tembo.io/tryFreeButton.svg" alt="Tembo Cloud Try Free">
-    </a>
-  </div>
-</p>
-
-</h1>
-
-A Postgres extension that automates the transformation and orchestration of text to embeddings and provides hooks into the most popular LLMs. This allows you to do vector search and build LLM applications on existing data with as little as two function calls.
+A Postgres extension that automates the transformation and orchestration of text to embeddings and provides hooks into the most popular LLMs. This allows you to perform vector search and build LLM applications on existing data with as little as two function calls.
 
 This project relies heavily on the work by [pgvector](https://github.com/pgvector/pgvector) for vector similarity search, [pgmq](https://github.com/tembo-io/pgmq) for orchestration in background workers, and [SentenceTransformers](https://huggingface.co/sentence-transformers).
 
 ---
 
-[![Static Badge](https://img.shields.io/badge/%40tembo-community?logo=slack&label=slack)](https://join.slack.com/t/tembocommunity/shared_invite/zt-277pu7chi-NHtvHWvLhHwyK0Y5Y6vTPw)
-[![PGXN version](https://badge.fury.io/pg/vectorize.svg)](https://pgxn.org/dist/vectorize/)
-[![OSSRank](https://shields.io/endpoint?url=https://ossrank.com/shield/3815)](https://ossrank.com/p/3815)
-
-pg_vectorize powers the [VectorDB Stack](https://tembo.io/docs/product/stacks/ai/vectordb) on [Tembo Cloud](https://cloud.tembo.io/) and is available in all hobby tier instances.
+`pg_vectorize` powers the [VectorDB Stack](https://tembo.io/docs/product/stacks/ai/vectordb) on [Tembo Cloud](https://cloud.tembo.io/) and is available in all hobby tier instances.
 
-**API Documentation**: https://tembo.io/pg_vectorize/
+**API Documentation**: [https://tembo.io/pg\_vectorize/](https://tembo.io/pg_vectorize/)
 
-**Source**: https://github.com/tembo-io/pg_vectorize
+**Source**: [https://github.com/tembo-io/pg\_vectorize](https://github.com/tembo-io/pg_vectorize)
 
 ## Features
 
-- Workflows for both vector search and RAG
-- Integrations with OpenAI's [embeddings](https://platform.openai.com/docs/guides/embeddings) and [chat-completion](https://platform.openai.com/docs/guides/text-generation) endpoints and a self-hosted container for running [Hugging Face Sentence-Transformers](https://huggingface.co/sentence-transformers)
+- Workflows for both vector search and RAG (Retrieval-Augmented Generation)
+- Integrations with OpenAI's [embeddings](https://platform.openai.com/docs/guides/embeddings) and [chat-completion](https://platform.openai.com/docs/guides/chat) endpoints, and a self-hosted container for running [Hugging Face Sentence-Transformers](https://huggingface.co/sentence-transformers)
```

Comment:
If you go to `https://platform.openai.com/docs/guides/chat`, it will forward you to `https://platform.openai.com/docs/guides/text-generation`.

![image](https://github.com/user-attachments/assets/9bbd0eb8-b1f7-4876-81f7-af05958d227d)